client/src/components/MainPage/Spectialization/index.jsx

```diff
@@ -91,10 +92,15 @@ class Specialization extends Component {
           </h1>
         )}
         <SpecializationCard values={items} />
-        {found && <p className="specialization__showmore">Show All </p>}
+        {found && <p className="specialization__showmore" />}
       </div>
     );
   }
 }
 
+Specialization.propTypes = {
+  history: PropTypes.isRequired,
```

Comment:
You could add a type here, too, for added safety. For example, PropTypes.object.isRequired.